package.json

```diff
@@ -4,13 +4,16 @@
   "version": "5.25.0",
   "description": "Utilities to help when developing terra modules.",
   "engines": {
-    "node": "10 || 12|| 14",
+    "node": "10 || 12 || 14",
     "npm": "6 || 9"
```

Comment:
Since workspaces is a npm 9 feature, we should remove npm 6.

Reply:
Done!